build.proj

```diff
@@ -33,6 +33,7 @@
       RestoreProjects;
       BuildNative;
       $(TraversalBuildDependsOn);
+      DownloadExternalTestFiles;
```

Comment:
Will this thing download a file on every build or only if the file is not present already?

Reply:
I'm trying to make it download file only once, but struggle for right now, see Eric comment and my reply.